remote_control.py

```diff
@@ -110,3 +117,13 @@ def constrain(self, val, min_val, max_val):
     def stop(self):
         logging.info("RC challenge stopping")
         self.killed = True
+
+    def dither(self, counter, speed):
+        if self.deadband < abs(speed) < self.slow_speed:
+            speed = int(speed - math.copysign(1, speed))
+            counter += 1
+            if counter < self.boost_cycles:
+                speed = speed + int(math.copysign(self.slow_speed, speed))
+            if counter > (self.boost_cycles + self.boost_dwell):

```

Comment:
`elif` here?  if `counter < self.boost_cycles`, then there's no point evaluating this condition.

Reply:
good point